components/aws/sagemaker/tests/integration_tests/resources/config/kmeans-mnist-hpo/config.yaml

```diff
@@ -0,0 +1,56 @@
+PipelineDefinition: resources/definition/hpo_pipeline.py
+TestName: kmeans-mnist-hpo-training
+Timeout: 3600
+ExpectedTrainingImage: ((KMEANS_REGISTRY)).dkr.ecr.((REGION)).amazonaws.com/kmeans:1
+Arguments:
+  region: ((REGION))
+  algorithm_name: K-Means
```

Comment:
I wonder if Nick is seeing that error because this needs to be a string. But it should have failed for you too in that case.

Reply:
It will be converted to string when it loads and i was unable to reproduce